phpBB/assets/javascript/editor.js

```diff
@@ -104,8 +105,8 @@ function bbfontstyle(bbopen, bbclose) {
 	}
 	// IE
 	else if (document.selection) {
-		var range = textarea.createTextRange(); 
-		range.move("character", new_pos); 
+		var range = textarea.createTextRange();
+		range.move("character", new_pos);
```

Comment:
Only place in this file where double quotes are used.
Eventhough eslinting is disabled, might be worth changing.
Or perhaps you would rather not alter it as it's someone else's script?